Toolkit/ArcGISToolkit/TimeSlider.swift

```diff
@@ -542,11 +542,24 @@ public class TimeSlider: UIControl {
     // MARK: Other Properties
     
     /**
-     This will initialize the slider's fullExtent, currentExtent and timeStepInterval
-     properties based on time enabled layers in the geo view.
+     The geo view used to initialize the time slider properties.
      */
     public private(set) var geoView: AGSGeoView?
     
+    /**
+     A Boolean value indicating whether changes (change in layers and time extent) in geo view are being observed by the time slider.
```

Comment:
"geo view" -> `geoView`;
Also, say "are being observed", seems to indicate a passive operation by the user.  I think this should be "should be observed", because it's something the user can set if they want to.

Reply:
fixed